public/css/search.css

```diff
@@ -1,26 +1,26 @@
-.search {
+/* formatting of the section containing the search box */
+.search-section {
+    display: inline-flex;
+    box-sizing: border-box;
     position: relative;
     max-width: 100%;
-    font-size: 100%;
-}
-
-/* set font of search bar and search input */
-.search {
-    font-family: Fjalla One;
+    background: white;
+    border-radius: 5px;
+    box-shadow: 0 0 20px rgba(0, 0, 0, 0.15);
 }
 
+/* search box formatting */
 .search {
-    display: inline-flex;
-    box-sizing: border-box;
+    border: none;
+    border-radius: 5px;
     width: 100%;
-    padding: 6px;
-    background: #eeeeee;
+    padding: 2px;
+    background: white;
+    font-size: 100%;
 }
 
-/* style inside search bar */
-.search input[type=text] {
-    background: white;
-    border-radius: 5px;
-    border: none;
-    width: 100%;
+/* style inside search bar, removes outline of the search box when selected */
+.search:focus {
+  outline: none;
```

Comment:
I think the search bar looks great now they're consistent, but why remove the outline on selection?

It lessens accessibility/usability (the idea of the outline is to show when an element is selected, because not everyone is using a mouse to navigate web pages - particularly people who can't).

Reply:
Just for the way it looked, hadn't considered the usability issue. Also didn't realise outline can be formatted 😅  so I've added that now and made the outline visible on selection.